eccp

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/python
+
+import ConfigParser
+import sys
+import StringIO
+from verify import *
+
+#oneseventwoeight = 1728
+
+with open(sys.argv[1], 'r+') as f:
+    config_buff = f.read()
+    f.close()
+config_ini = StringIO.StringIO(config_buff)
+
+config = ConfigParser.RawConfigParser(allow_no_value=True)
+#config.read(config_ini)
+config.readfp(config_ini)
+sections = config.sections()
+
+# dict with params
+
+def par_set():
+    global params
```

Comment:
I am using it in latter in the code, and also resetting with new call to this function, maybe I can find a more elegant solution for this.